jakarta.jdt/org.eclipse.lsp4jakarta.jdt.test/src/main/java/org/eclipse/lsp4jakarta/jdt/jsonb/JsonbDiagnosticsCollectorTest.java

```diff
@@ -71,4 +71,22 @@ public void deleteExtraJsonbCreatorAnnotation() throws Exception {
 
         assertJavaCodeAction(codeActionParams2, utils, ca2);
     }
+    
+    @Test
+    public void diagnosticWhenJsonbTransientIsNotMutuallyExclusive() throws Exception {
```

Comment:
consider renaming to shorten, maybe 'JsonbTransientNotMutuallyExclusive()`

Reply:
Done